lightpath/version.py

```diff
@@ -0,0 +1,47 @@
+"""
```

Comment:
Do I still need to fix the bug in this boilerplate file that makes it miss the repo-level `.git_archival.txt`?

Reply:
I'm not sure if we converged on a proper solution for that... Was it the result of the "let's fix the version 0.0.0 shown when building the conda package?" or something else?

Reply:
I think the fix you applied to pcdsdevices is the correct one

Reply:
Right, so correct me if I'm remembering incorrectly - the `0.0.0` fix was made by way of adding in setuptools-scm to the conda build requirements.

The addition of the `git_archival` clause in `version.py` came along at the same time in an attempt to fix that, but it also made us realize simultaneously that a release source code tarball on GitHub would not be properly versioned unless we modified that line.

I _think_ that's how it went. In that case, I agree that we should update the template and this `version.py` to include a check for `git_archival.txt`